doc/stdlib/math.rst

```diff
@@ -1757,7 +1757,10 @@ multi-threading. Use ``FFTW.set_num_threads(np)`` to use ``np`` threads.
 
    A multidimensional FFT simply performs this operation along each transformed dimension of ``A``\ .
 
-   Higher performance is usually possible with multi-threading. Use ``FFTW.set_num_threads(np)`` to use ``np`` threads, if you have ``np`` processors.
+   .. note::
+      * Julia starts FFTW up with 1 thread by default. Higher performance is usually possible by increasing number of threads. Use ``FFTW.set_num_threads(Sys.CPU_CORES)`` to use as many threads as cores on your system.
```

Comment:
this note now makes the above version where it says
```
By default, Julia does not use multi-threaded
 FFTW. Higher performance may be obtained by experimenting with
multi-threading. Use ``FFTW.set_num_threads(np)`` to use ``np`` threads.
```
redundant, so you can delete that one. Having this in the docstring is a little more useful.